devicetypes/fibargroup/fibaro-motion-sensor-zw5.src/fibaro-motion-sensor-zw5.groovy

```diff
@@ -1,315 +1,478 @@
+    logging("${device.displayName} - BatteryReport received, value: ${cmd.batteryLevel}", "info")
+    sendEvent(name: "battery", value: cmd.batteryLevel.toString(), unit: "%", displayed: true, isStateChange: true)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd) {
+    logging("${device.displayName} - SensorMultilevelReport received, sensor: ${cmd.sensorType}, scaledValue: ${cmd.scaledSensorValue}", "info")
+    switch (cmd.sensorType as Integer) {
+        case 1:
+            def cmdScale = cmd.scale == 1 ? "F" : "C"
+            sendEvent(name: "temperature", unit: getTemperatureScale(), value: convertTemperatureIfNeeded(cmd.scaledSensorValue, cmdScale, cmd.precision), displayed: true)
+            break
+        case 3:
+            sendEvent(name: "illuminance", value: cmd.scaledSensorValue.toInteger().toString(), unit:"lux", displayed: true)
+            break
+        case [25,52,53,54]:
+            motionEvent(cmd.sensorType, cmd.scaledSensorValue )
+            break
+    }
+}
+
+private motionEvent(Integer sensorType, value) {
+    logging("${device.displayName} - Executing motionEvent() with parameters: ${sensorType}, ${value}", "info")
+    def axisMap = [52: "yAxis", 53: "zAxis", 54: "xAxis"]
+    switch (sensorType) {
+        case 25:
+            sendEvent(name: "motionText", value: "Vibration:\n${value} MMI", displayed: false)
+            break
+        case 52..54:
+            sendEvent(name: axisMap[sensorType], value: value , displayed: false)
+            runIn(2,"axisEvent")
+            break
+    }
+}
+
+private axisEvent() {
+    logging("${device.displayName} - Executing axisEvent() values are: ${device.currentValue("xAxis")}, ${device.currentValue("yAxis")}, ${device.currentValue("zAxis")}", "info")
+    def xAxis = Math.round((device.currentValue("xAxis") as Float) * 100)
+    def yAxis = Math.round((device.currentValue("yAxis") as Float) * 100) // * 100 becaouse from what I can tell apps expect data in cm/s2
+    def zAxis = Math.round((device.currentValue("zAxis") as Float) * 100)
+    sendEvent(name: "motionText", value: "X: ${device.currentValue("xAxis")}\nY: ${device.currentValue("yAxis")}\nZ: ${device.currentValue("zAxis")}", displayed: false)
+    sendEvent(name: "threeAxis", value: "${xAxis},${yAxis},${zAxis}", isStateChange: true, displayed: false)
+}
+
+// Parameter configuration, synchronization and verification
+def updated() {
+    if ( state.lastUpdated && (now() - state.lastUpdated) < 500 ) return
+    
+    logging("${device.displayName} - Executing updated()","info")
+
+    if ( settings.tamperOperatingMode as Integer == 0 ) {
+        resetMotionTile()
+    }
+
+    syncStart()
+    state.lastUpdated = now()
+	configure()
+}
+
+def installed(){
+	sendEvent(name: "checkInterval", value: 1920, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	configure()
+}
+
+def configure() {

```

Comment:
if the calls in this method aren't actually configuring anything, they shouldn't be in `configure`